src/code/utils/translate.ts

```diff
@@ -32,22 +32,22 @@ interface LanguageFileEntry {
 }
 
 const languageFiles: LanguageFileEntry[] = [
-  {key: 'de',    contents: de},     // German
-  {key: 'el',    contents: el},     // Greek
-  {key: 'en-US', contents: enUS},   // US English
-  {key: 'es',    contents: es},     // Spanish
-  {key: 'fa',    contents: fa},     // Farsi (Persian)
-  {key: 'he',    contents: he},     // Hebrew
-  {key: 'ja' ,   contents: ja},     // Japanese
-  {key: 'ko' ,   contents: ko},     // Korean
-  {key: 'nb',    contents: nb},     // Norwegian Bokmål
-  {key: 'nn',    contents: nn},     // Norwegian Nynorsk
-  {key: 'pl',    contents: pl},     // Polish Polski
-  {key: 'pt-BR', contents: ptBR},   // Brazilian Portuguese
-  {key: 'th',    contents: th},     // Thai
-  {key: 'tr',    contents: tr},     // Turkish
-  {key: 'zh',    contents: zhHans}, // Simplified Chinese
-  {key: 'zh-TW', contents: zhTW}    // Traditional Chinese (Taiwan)
+  {key: 'de',       contents: de},     // German
+  {key: 'el',       contents: el},     // Greek
+  {key: 'en-US',    contents: enUS},   // US English
+  {key: 'es',       contents: es},     // Spanish
+  {key: 'fa',       contents: fa},     // Farsi (Persian)
+  {key: 'he',       contents: he},     // Hebrew
+  {key: 'ja' ,      contents: ja},     // Japanese
+  {key: 'ko' ,      contents: ko},     // Korean
+  {key: 'nb',       contents: nb},     // Norwegian Bokmål
+  {key: 'nn',       contents: nn},     // Norwegian Nynorsk
+  {key: 'pl',       contents: pl},     // Polish Polski
+  {key: 'pt-BR',    contents: ptBR},   // Brazilian Portuguese
+  {key: 'th',       contents: th},     // Thai
+  {key: 'tr',       contents: tr},     // Turkish
+  {key: 'zh-Hans',  contents: zhHans}, // Simplified Chinese
```

Comment:
There is something in the back of my brain that tells me we need to keep it just `zh`.  I think it is because we want any Chinese variant other than Taiwanese to default to Hans which it will do if it is just `zh`.  We don't have to do the same for `en-GB`/`en-CA` to use `en-US` etc because `en` is the default language used if the specific language requested by the browser is not used.

Reply:
I'll see if I can figure that out. Other things weren't working with switching due to the code mismatch between CODAP and CFM, so if you're right that `zh` needs special treatment then we may require some special-case code.

Reply:
I'll do the experiment, but I think it should be fine because both CODAP and CFM use the following code to add languages to the map:
```typescript
const translations: Record<string, LanguageFileContent> =  {}
languageFiles.forEach(function(lang) {
  translations[lang.key.toLowerCase()] = lang.contents
  // accept full key with region code or just the language code
  const baseLang = getBaseLanguage(lang.key)
  if (baseLang && !translations[baseLang]) {
    translations[baseLang] = lang.contents
  }
})
```
so `zh-Hans` will be added to the map under both `zh-Hans` and `zh` because the code specifies that in the case of multiple instances of the same language, the first one takes precedence.

Reply:
I just ran the experiment. If I set my preferred browser language to simply Chinese, I get `zh-Hans`.

Reply:
I remember what it was - this was something in SageModeler when it passed down the language to CODAP in the url.  We had to only give it the major language code, I think because CODAP only had `zh`. 

https://github.com/concord-consortium/sage-modeler-site/blob/master/src/code/codap-iframe-src.ts#L20-L24

So I guess no change is needed.